docs/_posts/plugins/0100-01-01-mapbox-gl-compare.html

```diff
@@ -8,6 +8,8 @@
   - isc
 code: https://github.com/mapbox/mapbox-gl-compare
 license: ISC
+suffix: compare
+css: true
+js: true
```

Comment:
We may need to go back to the explicit markup if we hever have a 3rd party plugin that:

 - requires extra boilerplate
 - is hosted in a different way 
 - uses a different naming convention